test/rest_request_template/travis_correct_auto_table_creation.json

```diff
@@ -2,7 +2,6 @@
   "name": "SNOWFLAKE_CONNECTOR_NAME",
   "config": {
     "connector.class": "com.snowflake.kafka.connector.SnowflakeSinkConnector",
-    "snowflake.role.name": "TESTROLE_KAFKA",
```

Comment:
you dont need this?

Reply:
Similarly for other tests?

Reply:
There's a field with the same name below. This was mistakenly introduced by me at some time.